packages/mobx/__tests__/v5/base/typescript-tests.ts

```diff
@@ -2054,6 +2054,17 @@ test("type inference of the action callback", () => {
     }
 })
 
+test("TS - type inference of Set", () => {
+    const set = observable.set<number>()
+    set.add(1)
```

Comment:
If the point of this line is to test the correct usage, please add the other methods as well `.has(1)`, `.delete(1)`.